client/components/StoryTime.js

```diff
@@ -100,6 +102,9 @@ class StoryTime extends React.Component {
     });
   }
 
+  backToLibrary(e) {
```

Comment:
this could be just an anonymous function on the click event in the component where invoked since we are not doing anything with state here... but it is okay like this for now